package.json

```diff
@@ -43,23 +43,24 @@
     "@nationalbankbelgium/eslint-config": "^14.0.0-rc.0",
     "@ng-idle/core": "^11.0.3",
     "@ng-idle/keepalive": "^11.0.3",
-    "@ngrx/effects": "^13.2.0",
-    "@ngrx/store": "^13.2.0",
-    "@ngrx/store-devtools": "^13.2.0",
+    "@ngrx/effects": "^14.3.3",
+    "@ngrx/store": "^14.3.3",
+    "@ngrx/store-devtools": "^14.3.3",
     "@ngtools/json-schema": "^1.1.0",
-    "@ngx-translate/core": "^13.0.0",
+    "@ngx-translate/core": "^14.0.0",
     "@npmcli/ci-detect": "^1.3.0",
     "@release-it/conventional-changelog": "^8.0.1",
     "@sqltools/formatter": "^1.2.3",
     "@types/jasmine": "~5.1.4",
     "@types/karma": "^6.3.8",
-    "@types/lodash-es": "^4.17.4",
-    "@types/node": "^18.19.4",
+    "@types/lodash": "4.14.202",
+    "@types/lodash-es": "4.17.12",
+    "@types/node": "18.19.15",
```

Comment:
Since the engine is `"node": ">=20.11.1"`, should not we use `"@types/node": "20.x.y"` ?